<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spi</class>
 <widget class="QWidget" name="Spi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="spi_lcdNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="spi_getButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spi_exitButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
